### DISCUSSION TITLE
Plugin Unavailable error while data source adding

### DISCUSSION THREAD

**Discussion by yatif** [grafana/grafana-infinity-datasource#374] [Q&A]
Hello there,

I am receiving "Plugin Unavailable" error when I press the "save&test" button on data sources section for grafana infinity datasource plugin. 
By the way I am using the grafana on my local computer with docker image. my grafana version is 9.0.0

Thank you

**Comment by yesoreyeram** [comment]
What's your docker compose looks like? Or what command you are running to start docker instance? How do I reproduce locally?
